client/src/styles/legacy/_forms.scss

```diff
@@ -150,6 +150,23 @@ fieldset {
       }
     }
   }
+
+  .title-tip.tip {
```

Comment:
This should be in a SCSS file in the same folder as Tip.

Reply:
Moved to its own scss file